lib/index.js

```diff
@@ -150,7 +150,10 @@ Builder.prototype.build = function(options) {
 	}
 
 	function fileHandler(file, currentFileInfo, handleDataAndCallCallback, callback) {
-		const pathname = path.posix.join(currentFileInfo.currentDirectory, file);
+		let pathname = path.posix.join(currentFileInfo.currentDirectory, file);
+		if (path.isAbsolute(file)) {
+			pathname = path.posix.join(file);
```

Comment:
what is this single join good for? isn't this just a normalize?

Reply:
and the join call in L153 is unnecessary for an absolute file, maybe better move it into an else branch.

Reply:
shouldn't it be path.posix.isAbsolute?

Reply:
> what is this single join good for? isn't this just a normalize?

good point, yes [join](https://nodejs.org/api/path.html#path_path_join_paths) with a single argument is a [normalize](https://nodejs.org/api/path.html#path_path_normalize_path)

Reply:
> shouldn't it be path.posix.isAbsolute?

done with 39df2b1

Reply:
LGTM. But I don't know the code enough for an approval.